image-customize

```diff
@@ -52,91 +51,91 @@ def prepare_install_image(base_image, install_image, resize):
     return install_image
 
 
-def run_command(machine_instance, commandlist):
-    """Run command inside image"""
-    for foo in commandlist:
+class ActionBase(argparse.Action):
+    '''Keep an ordered list of actions'''
+
+    def __call__(self, parser, namespace, value, option_string=None):
+        namespace.actions.append((self.execute, value))
+
+
+class InstallAction(ActionBase):
+    '''Install local rpm or distro package'''
+    @classmethod
```

Comment:
And that's still an open question.

Reply:
Ah, I didn't know about `@staticmethod`, that's fine of course. We don't need to access/store class attributes here. Changed.